BSP/Simulator/Hardware/Pedals.py

```diff
@@ -0,0 +1,31 @@
+import os
+import csv
```

Comment:
I would call this module `ADC.py` just to keep the naming parallel to the C stuff